src/components/GridLayout.vue

```diff
@@ -7,6 +7,10 @@
 <style scoped lang="scss">
 @import "~assets/styles/vars";
 @import "~assets/styles/grid";
+
+.gridded {
+  z-index: 4;
```

Comment:
Having `z-index: 4` here kind of breaks the component abstraction. That's not so great because in the future, a developer that needs to tweak this value may inadvertently break something else on the site.

Correct me if I'm wrong, but I think you added `z-index: 4` to make the GridLayout work with another component. (Connect page?) If that's the case, I think it'd be better to move the `z-index: 4` declaration to the Connect page.

Remember that you can apply classes to components you've imported elsewhere. Here's an example:

```vue
<template>
  <GridLayout class="grid-layout">
    <ComponentWithWeirdZIndex class="other" />
  </GridLayout>
</template>

<script>
import GridLayout from '~/components/GridLayout'
export default {
  components: { GridLayout }
}
</script>

<style scoped>
.grid-layout {
  z-index: 4;
}
.other {
  z-index: 3;
}
</style>
```

But that's just an example. It shows how you can keep classes with z-index properties in the same stacking context together. You don't have to do it exactly like that.

Reply:
It's actually something consistent on the site; every page has a layout of 
```
<layout>
  <navbar>
  <backgroundText/>
  <pageContent/>
</layout>
```
So I guess technically it should go in the `Default.vue` layout component(?)

Reply:
Yeah, I was noticing some of that after I made my comment. Yes, the default layout component makes sense then.